src/math/warpExposure.cc

```diff
@@ -158,6 +162,64 @@ std::string NearestWarpingKernel::NearestFunction1::toString(std::string const &
     return os.str();
 }
 
+namespace {
+
+struct LanczosKernelPersistenceHelper {
```

Comment:
When I implemented `StorableHelperPersistenceHelper`, I deleted the move and copy operations (which I'm sure I just copied from `PersistenceFactory`).  I don't know if that was strictly necessary then, but I thought I'd at least point this out so you can think about it.

Reply:
Yes, a singleton must be non-copyable. However, I don't really like the `*Helper` formalism (just too much boilerplate), and since this class is only reachable in this one file, I feel the readability benefits of skipping those bits are worth it.

Actually, by that argument perhaps these classes shouldn't attempt to be singleton-like at all.

Reply:
Great, thanks for the answer.  Minimizing boilerplate sounds good to me, including removing the singleton-ness if you like (as long as everything still tests fine of course).